src/hotspot/share/sanitizers/address.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1998, 2025, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please keep it as it is.

`XX, YY` means that this file was created at `XX` year and lastly updated at `YY` year. If `XX == YY` then use `XX`.

Reply:
No, @afshin-zafari was right, this was an oversight of me. The fix is fine.